modules/custom/az_publication/az_publication.install

```diff
@@ -8,6 +8,14 @@
 use Drupal\Core\Database\Database;
 use Drupal\Core\Utility\UpdateException;
 
+/**
+ * Implements hook_uninstall().
+ */
+function az_publication_uninstall() {
+  // Delete az_publication az_author auto entity label settings.
+  \Drupal::configFactory()->getEditable('auto_entitylabel.settings.az_author.az_author')->delete();
+}
+
```

Comment:
There is a hook_uninstall implementation here because `auto_entitylabel.settings.az_author.az_author` won't delete itself (possibly because it is simple config)  I do not know why the changes in https://github.com/az-digital/az_quickstart/pull/3034/files#diff-2d703919271e6fa13e5d2832238ef8c6dd1594adf2d2ae09218a9d9c48970ff8 didn't trigger this to uninstall.